src/mc_control/ControllerClient.cpp

```diff
@@ -699,7 +699,7 @@ void ControllerClient::handle_form(const ElementId & id, const mc_rtc::Configura
         form_array_input(id, name, required, el[3], el[4], el.size() > 5 ? el[5] : true);
         break;
       case Elements::ComboInput:
-        form_combo_input(id, name, required, el[3], el[4]);
+        form_combo_input(id, name, required, el[3], el[4], el[5], el[6]);
```

Comment:
This needs to check `el.size()` and provide the `-1` default  otherwise (this allows a client to work correctly if it is connected to an older version of mc_rtc)